React/Views/RCTPicker.m

```diff
@@ -23,16 +23,17 @@ @implementation RCTPicker
 - (instancetype)initWithFrame:(CGRect)frame
 {
   if ((self = [super initWithFrame:frame])) {
-    _color = [RCTUIColor blackColor]; // TODO(OSS Candidate ISS#2710739)
 #if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
     _font = [UIFont systemFontOfSize:21]; // TODO: selected title default should be 23.5
+    _color = [RCTUIColor blackColor]; // TODO(OSS Candidate ISS#2710739)
```

Comment:
A seemingly minor thing, but can we move this up a line so we match the order of these calls upstream? That will make future merges/resolving conflict much more straightforward